docker/jenkins/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+master:
```

Comment:
do we need this file? looks like we  already have that in docker/docker-compose.yml